### ISSUE TITLE
maybe add more movement to the tentacles rather than them moving around in ovals

### ISSUE THREAD

**Issue by humzahre** [nonamecrackers2/crackers-wither-storm-mod#1041]
***Please read the guidelines before submitting:*** https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/217

**Instead of the WS Tentacles moving around in ovals, add more movement to almost match the Real WS Tentacles**

https://user-images.githubusercontent.com/125754458/226056293-e24a61d2-dd3d-4d3c-aa08-a0bd01bfd7e8.mp4

**Comment by humzahre** [comment]
btw dont change the tentacle model itself, just add more movement to make it look cooler

**Comment by Globalfirestorm10** [comment]
Engender 💀

**Comment by GrandmaPork** [comment]
I think they’re perfect as is. Very ominous in the distance

**Comment by myron-obsolete-account** [comment]
dude what the hell dont come here with engender thats unrealistic as hell im sorry but no

**Comment by myron-obsolete-account** [comment]
also #47 close this now

**Comment by myron-obsolete-account** [comment]
and please start looking for issues with same/similar topics before just hitting the "new issue" button

**Comment by myron-obsolete-account** [comment]
thank you and have a nice day

**Comment by humzahre** [comment]
MoRoN more like Moron thats why ure so rude bc u a moron lol btw my desc changed

**Comment by humzahre** [comment]
> Engender 💀

You 💀💀 as The eternal flames of hellfire

**Comment by Globalfirestorm10** [comment]
> MoRoN more like Moron thats why ure so rude bc u a moron lol btw my desc changed

Says the guy who's so lazy he can't even take 5 seconds to fill out a template properly 💀

**Comment by Globalfirestorm10** [comment]
> > Engender 💀
> 
> 
> 
> You 💀💀 as The eternal flames of hellfire

So what? It's just a username, kid

**Comment by Globalfirestorm10** [comment]
> dude what the hell dont come here with engender thats unrealistic as hell im sorry but no
> 
> 

Based, and true

**Comment by humzahre** [comment]
> > MoRoN more like Moron thats why ure so rude bc u a moron lol btw my desc changed
> 
> Says the guy who's so lazy he can't even take 5 seconds to fill out a template properly 💀

Hahahaha.. i deserved that. 😒😒

**Comment by Globalfirestorm10** [comment]
> ***Please read the guidelines before submitting:*** https://github.com/nonamecrackers2/crackers-wither-storm-mod/issues/217
> 
> 
> 
> **Instead of the WS Tentacles moving around in ovals, add more movement to almost match the Real WS Tentacles**
> 
> 
> 
> https://user-images.githubusercontent.com/125754458/226056293-e24a61d2-dd3d-4d3c-aa08-a0bd01bfd7e8.mp4
> 
> 
> 
> 

Two things
1:regardless of whether you try to mask your cringe, we will always remember your reputation 

2. This is a duplicate, regardless of whether you edited it to mask the engender suggestion or not

**Comment by myron-obsolete-account** [comment]
> MoRoN more like Moron thats why ure so rude bc u a moron lol btw my desc changed

not meant to be rude at all but it starts to get annoying if people keep going on not to waste like 2 minutes of their lives to search for issues that might have their own topic and/or idea, and instead just fill the repository up with duplicate over duplicate. it's getting harder for cracker and co. to sort them all out, as already noticed on the issue with the body of the storm disappearing, where he sorted a different issue as a duplicate because he had to fly-read it, otherwise he couldnt catch up with all the new issues flooding in here

also the tentacle movements are already good as hell i mean would you find it logical to see a tentacle that weighs like 10 tons or so, to move really frickin' fast around the air? not me, at least. kinda getting over to personal opinions here but just to get your common sense baked up, the tentacle movements of the engender mod WS are not that realistic, plus the MCSM witherstorm's tentacle movements are nothing like the ones from the engender mod. also, CWSM's WS tentacle movements are similar to the movements of MCSM already. some scenes in MCSM have been partially rushed-animated thats why they move so fast (at least thats my guess). 

---

TL;DR: you are probably 5 years old

P.S.: you are the one being rude to me now by calling me moron duh. get defeated, kid

**Comment by myron-obsolete-account** [comment]
get common sens'd

ok i'm outta here before cracker shows up

**Comment by Globalfirestorm10** [comment]
Well said MoRon, also regarding MCSM: the animations for the storm and chapter one in general were very rushed, to the point that every few frames had some sort of visual error with it.

**Comment by myron-obsolete-account** [comment]
alright im about to drift out into zone off-topic but bro I really want someone to make a movie where the wither storm is a real life creature looking super realistic n stuff and people warriors or idk who try defeat it and the movie is like based on CWSM because the mod just has the most sense-making ways from summon to defeat of the WS. like, it was unclear how soren even made the super tnt in MCSM so i think cracker implemented a good build-around with the symbiont n stuff

why am i talking so much

**Comment by myron-obsolete-account** [comment]
most of the things I say don't even make any sense lmao

**Comment by humzahre** [comment]
pls dont be rude. *puts hands up*

**Comment by myron-obsolete-account** [comment]
not being rude at all. not on purpose, at least...

**Comment by Globalfirestorm10** [comment]
> alright im about to drift out into zone off-topic but bro I really want someone to make a movie where the wither storm is a real life creature looking super realistic n stuff and people warriors or idk who try defeat it and the movie is like based on CWSM because the mod just has the most sense-making ways from summon to defeat of the WS. like, it was unclear how soren even made the super tnt in MCSM so i think cracker implemented a good build-around with the symbiont n stuff
> 
> 
> 
> why am i talking so much

_GlobalFirestorm10 had a stroke trying to read this and died_

**Comment by Globalfirestorm10** [comment]
> pls dont be rude. *puts hands up*

How hypocritical

**Comment by myron-obsolete-account** [comment]
> _GlobalFirestorm10 had a stroke trying to read this and died_

I am truly sorry for this inconvenience. May I offer you an egg in these trying times?

**Comment by Globalfirestorm10** [comment]
> > > MoRoN more like Moron thats why ure so rude bc u a moron lol btw my desc changed
> 
> > 
> 
> > Says the guy who's so lazy he can't even take 5 seconds to fill out a template properly 💀
> 
> 
> 
> Hahahaha.. i deserved that. 😒😒

Yes, you did

**Comment by Globalfirestorm10** [comment]
> > _GlobalFirestorm10 had a stroke trying to read this and died_
> 
> 
> 
> I am truly sorry for this inconvenience. May I offer you an egg in these trying times?

What type? And sure, why not?

**Comment by myron-obsolete-account** [comment]
> What type? And sure, why not?

Ostrich egg.

**Comment by Globalfirestorm10** [comment]
> > What type? And sure, why not?
> 
> 
> 
> Ostrich egg.

My favorite, have ''em every day. Thanks for the offer, I'll take it

**Comment by FrozenDragon0** [comment]
> MoRoN more like Moron thats why ure so rude bc u a moron lol btw my desc changed

Jesus Christ you're so immature.

**Comment by humzahre** [comment]
ITS GOTTA BE PERFECT! ITS GOTTA BE PERFECT! ITS GOTTA BE P̸̡͓̞̠̪̯̳̳̩̦̹̠̠͌͗̒̇̈́̾̂͆̑̆͂̏̄͘̕͜͝ͅÉ̵͖̳̠̟͎̠̄ͅŖ̷̱̭̻̪̼̺̇͜F̴̧͈͖̜̟̳̬̯̝͔͉̮̈́͑̌̂́̅͘E̶̫͋̏̀͌̃̕͝Ć̸̡̙̻̣̟͇̭͙̪͔̠͜ͅT̷̨̨̨̬̯̣̼̬̲̞̱̭̗̲̳̯̼̪͆̽̔̉̎̏͗̅̈̅̇̀̓̏̍͐̾̕͝!̸̨͌̋͆͒̀͛̈́̉̆͑̍̋̒̕̚͠

**Comment by FrozenDragon0** [comment]
> ITS GOTTA BE PERFECT! ITS GOTTA BE PERFECT! ITS GOTTA BE P̸̡͓̞̠̪̯̳̳̩̦̹̠̠͌͗̒̇̈́̾̂͆̑̆͂̏̄͘̕͜͝ͅÉ̵͖̳̠̟͎̠̄ͅŖ̷̱̭̻̪̼̺̇͜F̴̧͈͖̜̟̳̬̯̝͔͉̮̈́͑̌̂́̅͘E̶̫͋̏̀͌̃̕͝Ć̸̡̙̻̣̟͇̭͙̪͔̠͜ͅT̷̨̨̨̬̯̣̼̬̲̞̱̭̗̲̳̯̼̪͆̽̔̉̎̏͗̅̈̅̇̀̓̏̍͐̾̕͝!̸̨͌̋͆͒̀͛̈́̉̆͑̍̋̒̕̚͠


Can you like stop whining? You're literally embarrassing yourself acting like this.

**Comment by myron-obsolete-account** [comment]
> > ITS GOTTA BE PERFECT! ITS GOTTA BE PERFECT! ITS GOTTA BE P̸̡͓̞̠̪̯̳̳̩̦̹̠̠͌͗̒̇̈́̾̂͆̑̆͂̏̄͘̕͜͝ͅÉ̵͖̳̠̟͎̠̄ͅŖ̷̱̭̻̪̼̺̇͜F̴̧͈͖̜̟̳̬̯̝͔͉̮̈́͑̌̂́̅͘E̶̫͋̏̀͌̃̕͝Ć̸̡̙̻̣̟͇̭͙̪͔̠͜ͅT̷̨̨̨̬̯̣̼̬̲̞̱̭̗̲̳̯̼̪͆̽̔̉̎̏͗̅̈̅̇̀̓̏̍͐̾̕͝!̸̨͌̋͆͒̀͛̈́̉̆͑̍̋̒̕̚͠
> 
> Can you like stop whining? You're literally embarrassing yourself acting like this.

I think he is referring to an upcoming SMG4 movie called "It's gotta be perfect" but that somehow makes this even worse. Not because I don't like SMG4 but because using random referrences that make no sense in this conversation while already having a really bad reputation here is just so ultimatively cringe.

**Comment by myron-obsolete-account** [comment]
I'm starting to wonder about 2 things:
1. can you block people in repositories so they can't open issues and chat anymore?
2. how old is humzahre even he must be like 5 years old lmao

**Comment by FrozenDragon0** [comment]
> I'm starting to wonder about 2 things:
> 
> 1. can you block people in repositories so they can't open issues and chat anymore?
> 2. how old is humzahre even he must be like 5 years old lmao

You can ban specific members from a github repo but I believe only Cracker's can do that.

**Comment by myron-obsolete-account** [comment]
> You can ban specific members from a github repo but I believe only Cracker's can do that.

now that's some good news!

**Comment by humzahre** [comment]
> > > ITS GOTTA BE PERFECT! ITS GOTTA BE PERFECT! ITS GOTTA BE P̸̡͓̞̠̪̯̳̳̩̦̹̠̠͌͗̒̇̈́̾̂͆̑̆͂̏̄͘̕͜͝ͅÉ̵͖̳̠̟͎̠̄ͅŖ̷̱̭̻̪̼̺̇͜F̴̧͈͖̜̟̳̬̯̝͔͉̮̈́͑̌̂́̅͘E̶̫͋̏̀͌̃̕͝Ć̸̡̙̻̣̟͇̭͙̪͔̠͜ͅT̷̨̨̨̬̯̣̼̬̲̞̱̭̗̲̳̯̼̪͆̽̔̉̎̏͗̅̈̅̇̀̓̏̍͐̾̕͝!̸̨͌̋͆͒̀͛̈́̉̆͑̍̋̒̕̚͠
> > 
> > 
> > Can you like stop whining? You're literally embarrassing yourself acting like this.
> 
> I think he is referring to an upcoming SMG4 movie called "It's gotta be perfect" but that somehow makes this even worse. Not because I don't like SMG4 but because using random referrences that make no sense in this conversation while already having a really bad reputation here is just so ultimatively cringe.

🤨 HOW did you know?

**Comment by humzahre** [comment]
Like the moron requested, i will close this

**Comment by humzahre** [comment]
btw LucasDotje's WS mod is WAY BETTER THan cwsm even though it still WIP

https://www.youtube.com/watch?v=ZnmEslpHGhg

**Comment by myron-obsolete-account** [comment]
> btw LucasDotje's WS mod is WAY BETTER THan cwsm even though it still WIP
> 
> https://www.youtube.com/watch?v=ZnmEslpHGhg

![image](https://user-images.githubusercontent.com/53058491/226107760-de3c6363-1b37-462d-9296-2d1bdb95694b.png)
nah bro look at this goofy ahh octupus

**Comment by myron-obsolete-account** [comment]
> Like the moron requested, i will close this

you are very funny. you said i was mean to you yesterday but you called me moron twice now. something smells fishy, eh?


> 🤨 HOW did you know?

because I know where you live.

**Comment by FrozenDragon0** [comment]
> btw LucasDotje's WS mod is WAY BETTER THan cwsm even though it still WIP
> 
> https://www.youtube.com/watch?v=ZnmEslpHGhg

Bet you're only saying that because this mod isn't the so called "perfect" mod you want it to be, get over it. Just because a mod is based off a game doesn't mean it needs to be 100% accurate to said game.

**Comment by myron-obsolete-account** [comment]
cracker  spent over a year making this masterpiece and you are just straight up insulting him, humzahre, so do everyone here, including yourself, a massive favour, and get the hell out of here instead of crying all over the place how bad this mod is and everything. i can't believe these kinds of people still exist here, i thought it was finally over after i saw someone like you a few months ago.

**Comment by Globalfirestorm10** [comment]
> btw LucasDotje's WS mod is WAY BETTER THan cwsm even though it still WIP
> 
> 
> 
> https://www.youtube.com/watch?v=ZnmEslpHGhg

cry about it

**Comment by Globalfirestorm10** [comment]
> cracker  spent over a year making this masterpiece and you are just straight up insulting him, humzahre, so do everyone here, including yourself, a massive favour, and get the hell out of here instead of crying all over the place how bad this mod is and everything. i can't believe these kinds of people still exist here, i thought it was finally over after i saw someone like you a few months ago.

Exactly! Do us all a favor Humzahre,get off of discord, and tasks your whiny attitude elsewhere. 5 year olds don't belong here, nor anywhere else on apps like discord

**Comment by humzahre** [comment]
NOT A 5YROLD IM 11

**Comment by humzahre** [comment]
> > btw LucasDotje's WS mod is WAY BETTER THan cwsm even though it still WIP
> > https://www.youtube.com/watch?v=ZnmEslpHGhg
> 
> ![image](https://user-images.githubusercontent.com/53058491/226107760-de3c6363-1b37-462d-9296-2d1bdb95694b.png) nah bro look at this goofy ahh octupus

STILL IN DEVELOPMENT

**Comment by humzahre** [comment]
YOU KNOW WHAT STOP JUST... stop. leave me alone, leave this issue alone

**Comment by humzahre** [comment]
im done arguing.

**Comment by myron-obsolete-account** [comment]
> NOT A 5YROLD IM 11

thats worse than 5

**Comment by humzahre** [comment]
i said stop

**Comment by myron-obsolete-account** [comment]
> i said stop

just dont reply then lmao

**Comment by nonamecrackers2** [comment]
Everyone, GitHub is not the place to have conversations, my Discord server is for that.

@humzahre, You're obviously showing that you're not mature enough to handle being in this repository.

@Globalfirestorm10 @MoRoN222, Please be respectful to everyone regardless and not hurl insults. You guys obviously contributed to escalating the situation, and this is your only warning.

@MoRoN222, I appreciate what you're doing helping out, I really do, but let's be more respectful and nice about it. The way you point out duplicates to others or provide your opinion on ideas gives off a demeaning, almost insulting vibe, and I'd appreciate if you could be more respectful about it. Also, duplicates aren't federal offenses, there's a lot of them and it can be hard to search for the right terms. I should be more clear about duplicates in the guidelines, which is my fault. Thank you (and again, thank you for helping out)

**Comment by Globalfirestorm10** [comment]
> Everyone, GitHub is not the place to have conversations, my Discord server is for that.
> 
> 
> 
> @humzahre, You're obviously showing that you're not mature enough to handle being in this repository.
> 
> 
> 
> @Globalfirestorm10 @MoRoN222, Please be respectful to everyone regardless and not hurl insults. You guys obviously contributed to escalating the situation, and this is your only warning.
> 
> 
> 
> @MoRoN222, I appreciate what you're doing helping out, I really do, but let's be more respectful and nice about it. The way you point out duplicates to others or provide your opinion on ideas gives off a demeaning, almost insulting vibe, and I'd appreciate if you could be more respectful about it. Also, duplicates aren't federal offenses, there's a lot of them and it can be hard to search for the right terms. I should be more clear about duplicates in the guidelines, which is my fault. Thank you (and again, thank you for helping out)

Fair enough, my apologies. I guess I did overreact a bit in this situation. My only intention is to keep the repositories clear of duplicates and spam, but I guess I'll be a little more relaxed on that

**Comment by nonamecrackers2** [comment]
It's alright, just want this to be a welcoming place for people to submit there ideas

**Comment by Globalfirestorm10** [comment]
Fair enough, I understand. Unfortunately, there is no other way (as of now) for me and moron to communicate except for GitHub, which I believe you pointed out was one of the problems, as me and moron got off topic here.

**Comment by myron-obsolete-account** [comment]
man I'm dead

**Comment by Globalfirestorm10** [comment]
> man I'm dead

?

**Comment by myron-obsolete-account** [comment]
> ?

i overreacted too much

**Comment by Globalfirestorm10** [comment]
Ah I see

**Comment by myron-obsolete-account** [comment]
this issue has the most comments and I don't even know anymore how the hell we achieved that

**Comment by Globalfirestorm10** [comment]
> this issue has the most comments and I don't even know anymore how the hell we achieved that

Hey moron, we're famous on the CWSM discord. It's a shame that people like grandmapork hate us though :/

**Comment by myron-obsolete-account** [comment]
> Hey moron, we're famous on the CWSM discord. It's a shame that people like grandmapork hate us though :/

ah heell naw not the popularity

**Comment by myron-obsolete-account** [comment]
cough

yeah now I see what my cringe past-me did 💀
